app/components/Channels/ChannelCardListItem.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Box as BaseBox, Flex as BaseFlex } from 'rebass'
+import { Bar, Card, DataRow, Heading, Link, Text } from 'components/UI'
+import { ChannelCapacity, ChannelMoreButton, ChannelStatus } from 'components/Channels'
+import { blockExplorer } from 'lib/utils'
+import { Truncate } from 'components/Util'
+import withEllipsis from 'components/withEllipsis'
+import styled from 'styled-components'
+import { opacity, height } from 'styled-system'
+
+const ClippedHeading = withEllipsis(Heading.h1)
+const ClippedText = withEllipsis(Text)
+const Box = styled(BaseBox)(opacity)
+const Flex = styled(BaseFlex)(opacity, height)
+
+const ChannelCardListItem = ({
+  isAvailable,
+  channelId,
+  channelName,
+  channelPubKey,
+  channelPubKeyShort,
+  channelFundingTxid,
+  numUpdates,
+  csvDelay,
+  localBalance,
+  remoteBalance,
+  status,
+  showChannelDetail,
+  networkInfo,
+  ...rest
+}) => {
+  const opacity = isAvailable ? 1 : 0.3
+
+  return (
+    <Card {...rest}>
+      <Box as="header">
+        <Flex justifyContent="space-between">
+          <ClippedHeading my={1} opacity={opacity}>
+            {channelName}
+          </ClippedHeading>
+
+          <ChannelStatus status={status} mb="auto" />
+        </Flex>
+        <Flex justifyContent="space-between" opacity={opacity}>
+          <Text fontWeight="normal">Remote PubKey</Text>
```

Comment:
Looks like `Remote PubKey` should be made intl compatible

Reply:
Completely forgot to go back and extract these. Updated, thanks!